etc/notebooks/got_scotch_demo/scotch_dashboard.ipynb

```diff
@@ -429,7 +459,11 @@
     "urth_section": {}
    },
    "source": [
-    "## Dashboard Setup"
+    "## Dashboard Setup <span style=\"float: right; font-size: 0.5em\"><a href=\"#Got-Scotch?\">Top</a></span>\n",
+    "\n",
+    "Finally, we will set up the dashboard to show a short introduction line which scotch you're examining, then a scotch dropdown picker, a recommendation table and a radar chart to show how similar it is to your selection.\n",
```

Comment:
"... a short introduction line about which ..."

"... a recommendation table, and a ..."